### PR TITLE
Added 'player' npc

### PR DESCRIPTION
**Pull request by Slohive**
Added 'player' data to the JSON to represent a debug NPC that we want to create to help automate testing of the Simulation. The debug command will default 'Off", with the Player controlling their own Avatar. Turning the debug command 'On' will assume control of the NPC and run them as an NPC.

We have not created the debug command yet, the JSON is intended to pre-populate that data for engineering to hook into at a later date

### CODE REVIEW

**Comment by Slohive** [comment]
Added 'player' data to the JSON to represent a debug NPC that we want to create to help automate testing of the Simulation. The debug command will default 'Off", with the Player controlling their own Avatar. Turning the debug command 'On' will assume control of the NPC and run them as an NPC. 

We have not created the debug command yet, the JSON is intended to pre-populate that data for engineering to hook into at a later date

**Comment by tpower3** [comment]
> Added 'player' data to the JSON to represent a debug NPC that we want to create to help automate testing of the Simulation. The debug command will default 'Off", with the Player controlling their own Avatar. Turning the debug command 'On' will assume control of the NPC and run them as an NPC.
> 
> We have not created the debug command yet, the JSON is intended to pre-populate that data for engineering to hook into at a later date

This comment looks like it was intended to be the PR description. Comments can easily be lost on the timeline if the PR has lots of activity. Recommend copying your comment into the description.

File: Source/Atj/data/mvp_scenario.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "npcs": [
+        "player",
```
Comment (tpower3) [inline Source/Atj/data/mvp_scenario.json]: I asked you to put this here, but on second thought, lets remove it. The "player" implicitly exists, there's no reason to require you to specify them in the scene.

File: Source/Atj/data/mvp_scenario.json [hunk 2]
```diff
@@ -446,6 +494,81 @@
...
+                {
+                    "type": "behavior",
+                    "sync_time": "0.0",
+                    "behavior": "move_to",
+                    "target": "storage_cabinet_1"
+                },
+                {
+                    "type": "execute_action",
+                    "sync_time": "20.0",
+                    "action": "bind_player_to_debug_player_move_to_zaks_desk_routine_action"
```
Comment (tpower3) [inline Source/Atj/data/mvp_scenario.json]: Should this be changed to (stars excluded):
`bind_player_to_debug_player_move_to_zaks_desk**_1_**routine_action`?

**Comment by tpower3** [comment]
We discussed "reference_points" offline. The idea is that we can make "invisible" objects that can be walked through. These can be used to specify "move_to" locations.

We don't have these yet, and if we don't need them, there's no reason to add the complexity. But if they're narratively needed/useful, I don't see them being much work to support.

**Comment by Slohive** [comment]
Removed 'player' from NPC list. Added object reference (**1**) to bulletin board & zaks desk routines, to bring consistency. When referencing the object, always include the number at the end. Decided to forego the need for reference points at this time. 

This is ready for Final Review

File: Source/Atj/data/mvp_scenario.json (hunk 2 above)
Comment (tpower3) [reply Source/Atj/data/mvp_scenario.json]: This still needs to be updated @Slohive

**Comment by tpower3** [comment]
> Added object reference (1) to bulletin board & zaks desk routines, to bring consistency
In future PR's lets make changes like this in their own PR. It's unrelated to "Adding the player npc". It's fine for now though.

Otherwise, please make sure to address or resolve my open feedback.

**Review by tpower3 — APPROVED** [review]
LGTM

**Pull request merged**